cookiecutter.json

```diff
@@ -16,7 +16,7 @@
   "__prompts__": {
     "author_given_names": "Given name(s) of package author",
     "author_family_names": "Family name(s) of package author",
-    "author_email": "Email address for package author - will be part of package metadata.",
```

Comment:
Removed this period just for consistency with lack of punctuation on prompts for other (non-boolean) options